### PR TITLE
feat: upgrade Appsflyer SDK version to v6.14.5

### PR DESCRIPTION
**Pull request by shwetalodhe123**
# Description

- update version number to latest 6.14.5.
- testing using SPM and pod is completed.
- version update is tested on objC and swift sample apps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ For more information on configuring AppsFlyer as a destination in RudderStack an
 2. `RudderAppsFlyer` is available through [CocoaPods](https://cocoapods.org). To install it, add the following line to your `Podfile`:
 
 ```ruby
-pod 'RudderAppsFlyer', '~> 1.2.0'
+pod 'RudderAppsFlyer', '~> 1.2.1'
```
Comment (1abhishekpandey) [inline README.md:42]: It should be `1.3.0`, as the PR title is of type `feat`:
```suggestion
pod 'RudderAppsFlyer', '~> 1.3.0'
```
(If the PR is of type `fix` then we can update the patch version.)

File: RudderAppsFlyer.podspec [hunk 2]
```diff
@@ -1,24 +1,23 @@
 Pod::Spec.new do |s|
     s.name             = 'RudderAppsFlyer'
-    s.version          = '1.2.0'
+    s.version          = '1.2.1'
```
Comment (1abhishekpandey) [inline RudderAppsFlyer.podspec:3]:
```suggestion
    s.version          = '1.3.0'
```

File: RudderAppsFlyer.podspec [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
     s.homepage         = 'https://github.com/rudderlabs/rudder-integration-appsflyer-swift'
-    s.license          = { :type => "Apache", :file => "LICENSE" }
+    s.license          = { :type => "Apache", :file => "LICENSE.md" }
     s.author           = { 'RudderStack' => 'arnab@rudderlabs.com' }
     s.source           = { :git => 'https://github.com/rudderlabs/rudder-integration-appsflyer-swift.git' , :tag => "v#{s.version}"}
     
     s.ios.deployment_target = '13.0'
-    s.osx.deployment_target   = '10.13'
-    s.tvos.deployment_target  = '11.0'
+
```
Comment (1abhishekpandey) [inline RudderAppsFlyer.podspec:15]: Why have we removed these these two lines:
```
s.osx.deployment_target   = '10.13'
s.tvos.deployment_target  = '11.0'
```

File: Package.swift [hunk 4]
```diff
@@ -14,7 +14,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(name: "AppsFlyerLib-Static", url: "https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static", "6.14.0"..<"6.14.1"),
+        .package(name: "AppsFlyerLib-Static", url: "https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static", "6.14.5"..<"7.0.0"),
```
Comment (1abhishekpandey) [inline Package.swift:17]: Version restriction should be the same as the Podfile:
```suggestion
        .package(name: "AppsFlyerLib-Static", url: "https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static", "6.14.5"..<"6.14.6"),
```

File: README.md (hunk 1 above)
Comment (shwetalodhe123) [reply README.md]: Done

File: Package.swift (hunk 4 above)
Comment (shwetalodhe123) [reply Package.swift:17]: while adding package dependancies it was showing above version comparison at package window.

**Review by 1abhishekpandey — APPROVED** [review]

**Review by 1abhishekpandey — APPROVED** [review]

**Pull request merged**